common/djangoapps/util/tests/test_log_sensitive.py

```diff
@@ -0,0 +1,18 @@
+"""
+Tests for util.logging
+"""
+
+from common.djangoapps.util.log_sensitive import decrypt_log_message, encrypt_for_log, generate_reader_keys
+
+
+def test_round_trip():
```

Comment:
1. Consider `test_encryption_round_trip`
2. Note: this should fail without the module first installed. Lots of cancellation failure on unit tests right now. I hope that was a temporary issue, and am not clear if rebasing is required for fixing or not.

Reply:
1. Sure.
2. Unit test failures were because I couldn't compile or upgrade requirements, so I had to just `pip install PyNaCl` for local testing. (Resolved now.)